src/js/hooks/list-pages/productSupplier/useProductSupplierActions.jsx

```diff
@@ -17,6 +20,15 @@ const useProductSupplierActions = ({ fireFetchData }) => {
   const deleteProductSupplier = async (onClose, productSupplierId) => {
     try {
       await productSupplierApi.deleteProductSupplier(productSupplierId);
+      notification(NotificationType.SUCCESS)({
+        message: translate({
+          id: 'react.productSupplier.deleted.label',
+          defaultMessage: `Product Source ${productSupplierId} deleted`,
+          data: {
+            id: productSupplierId,
```

Comment:
wow, I've never seen this. :clap:

Reply:
Can you elaborate on that? What does "this" refer to?

Reply:
@jmiranda "this" refers to using `data` property of `transalate` method. I haven't known we could pass custom params for translation on the react side and then use it in the messages.properties like he did:
```groovy
react.productSupplier.deleted.label=Product Source ${id} deleted
```